tests/regression/gmail/test_firefox_gmail_ail_type_in_reply_field.sikuli/test_firefox_gmail_ail_type_in_reply_field.py

```diff
@@ -39,13 +39,13 @@
 app.wait_for_loaded()
 
 # Wait for stable
-sleep(5)
+sleep(2)
 
 # PRE ACTIONS
 app.click_first_mail()
```

Comment:
Well, I got your point now.

The real problem point is
```python
app.click_reply_btn()
sleep(3)
 
# Record T1, and capture the snapshot image
# Input Latency Action
t1 = time.time()
screenshot = capture(0, 0, capture_width, capture_height)
 
com.system_print('[log]  TYPE "a"')
type('a')
```

We should wait the `send btn` exist then type a.
It seems like I can add a `_il_type(message, wait_component=None)` method for typing cases.

👍